.circleci/config.yml

```diff
@@ -248,12 +248,16 @@ jobs:
       - run:
           name: Login to cloud.gov
           command: cf7 login -u ${CRT_USERNAME_DEV} -p ${CRT_PASSWORD_DEV} -o doj-crtportal -s ${CF_SPACE}
+      - acquire-lock:
+          app: crt-portal-django
       - run:
           name: Deploy clamav-rest to dev
           command: cf7 push clamav-rest --strategy rolling -f ${CF_MANIFEST}
       - run:
           name: Deploy crt-portal-django to dev
           command: cf7 push crt-portal-django --strategy rolling -f ${CF_MANIFEST}
```

Comment:
This is the crux of the code - since these are right next to each other, I've just locked them both under the portal app to keep things simple.